c2corg_ui/static/js/documentediting.js

```diff
@@ -162,6 +161,9 @@ app.DocumentEditingController = function($scope, $element, $attrs,
       this.api_.readDocument(this.module_, this.id_, this.lang_, true).then(
           this.successRead.bind(this)
       );
+    // new doc lang = user interface lang
```

Comment:
Move the comment inside the next block, else it is confusing to which block the comment is related to.